react/src/mocks/browser.ts

```diff
@@ -1,5 +1,7 @@
+/* eslint-disable no-unused-vars */
```

Comment:
Please delete the comments, imports and the lines that are offensive for eslint, we can pull that from git history.

Reply:
@pylipp I should then delete the file since we no longer use any of the embedded libraries